src/views/players/PlayerList.jsx

```diff
@@ -0,0 +1,53 @@
+import { useState, useEffect } from 'react';
+import { NavLink } from 'react-router-dom';
+import { deletePlayerById, getPlayers } from '../../services/players';
+import '../teams/teamdetail.css';
+
+function PlayerList(){
+    const [players, setPlayers] = useState([])
+    const [loading, setLoading] = useState(true)
+
+    const loadPlayers = async() => {
+        setLoading(true);
+        const playerList = await getPlayers();
+            setPlayers(playerList)
+            setLoading(false)
+    }
+
+    useEffect(() => {
+        loadPlayers();
+    }, [])
+
+    const handlePlayerDelete = async({id, name}) => {
```

Comment:
If a user tries to delete a team with players associated, an error happens. You'll want to either give the user a message letting them know they can't delete a team with players associated OR when the team is deleted, also delete the players associated with that team.